storm-core/src/jvm/org/apache/storm/container/cgroup/Device.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.container.cgroup;
+
+/**
+ * a class that represents a device in linux
+ */
+public class Device {
+
+    public final int major;
+    public final int minor;
+
+    public Device(int major, int minor) {
+        this.major = major;
+        this.minor = minor;
+    }
+
+    public Device(String str) {
+        String[] strArgs = str.split(":");
+        this.major = Integer.valueOf(strArgs[0]);
+        this.minor = Integer.valueOf(strArgs[1]);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
we are not using lombok annotations in storm currently. Perhaps we can int the future, but perhaps @abhishekagarwal87 can file a jira to investigate whether we should use it